### ISSUE TITLE
Some problems

### ISSUE THREAD

**Issue by vanderalex** [ogkalu2/comic-translate#24]
Unfortunately, the utility in its current form is barely used. 1. Some bubbles are cleaned badly. 2. Some text outside the bubbles (e.g. at the bottom of the page) is not recognised. 3. Recognition through GPT Vision works, but translation through it gives an error. But the main problem is that any recognition error even in one character leads to translation errors. 4. It is absolutely necessary to have a text editor with a 5. glossary, 6. text conversion to lower case (so the translation is more correct), with 7. auto-replacement of problematic characters (for example, words containing İ are not translated). Otherwise, there are dozens of recognition and translation errors per page, and without a text editor, the vast majority of comics remain unreadable.

**Comment by YasperMog1ot** [comment]
i agree with you, at least i need some settings to add white stroke outside the font
